### ISSUE TITLE
Animate slidable like dismiss does, but without calling dismiss

### ISSUE THREAD

**Issue by emadd** [letsar/flutter_slidable#112]
I have slidables that have two actions: archive and delete.  When they choose to delete, I call dismiss and it animates away, etc.  However, I would love to be able to perform the same animation when they choose archive without calling dismiss.  Maybe I'm just missing something?

**Comment by abraiyan** [comment]
I want this solution too

**Comment by letsar** [comment]
Ok I see, but the animation will end up with Archive in taking all the place. Is this what you want?

**Comment by theklausster** [comment]
onWillDismiss: (actionType) async {
          // do stuff like archive 
          return Future.value(false); // don't dismiss
        },

When I do this there's no animation applied, so the list tile flicks back to the state it was in before swiping.

Is this the same thing you see @emadd?

**Comment by theklausster** [comment]
Solved it by adding a key to the Slidable

**Comment by avdosev** [comment]
I am not an emadd, but I would like to see it like this

![slidable (1)](https://user-images.githubusercontent.com/39769345/99402814-3376da80-2903-11eb-8483-a13fafca19c4.gif)

**Comment by lironiz** [comment]
> I have slidables that have two actions: archive and delete. When they choose to delete, I call dismiss and it animates away, etc. However, I would love to be able to perform the same animation when they choose archive without calling dismiss. Maybe I'm just missing something?

Where and how do you call 'dismiss'? I'm trying to achieve an animated dismissal, but only after the user tapped on 'delete'. did you manage to do that?

**Comment by letsar** [comment]
@emadd can you add more details on what you want to be able to do?
@avdosev this is a different use case, can you open a new issue to track it?

**Comment by letsar** [comment]
This thread has been closed since there has not been any response for a while. If you are still experiencing a similar issue, please open a new bug.
